akka-actor-tests/src/test/scala/akka/actor/dispatch/ActorModelSpec.scala

```diff
@@ -399,7 +397,10 @@ abstract class ActorModelSpec(config: String) extends AkkaSpec(config) with Defa
 
                   System.err.println("Teammates left: " + team.size + " stopLatch: " + stopLatch.getCount + " inhab:" + dispatcher.inhabitants)
                   team.toArray sorted new Ordering[AnyRef] {
-                    def compare(l: AnyRef, r: AnyRef) = (l, r) match { case (ll: ActorCell, rr: ActorCell) ⇒ ll.self.path compareTo rr.self.path }
+                    def compare(l: AnyRef, r: AnyRef) = (l, r) match {
+                      case (ll: ActorCell, rr: ActorCell) ⇒ ll.self.path compareTo rr.self.path
+                      case (ll: AnyRef, rr: AnyRef)       ⇒ ll.hashCode compareTo rr.hashCode
+                    }
```

Comment:
Maybe:
```
                  import scala.collection.JavaConverters._
                  team.asScala.toList
                    .sortBy(_.self.path)
                    .foreach { cell: ActorCell ⇒
                      System.err.println(" - " + cell.self.path + " " + cell.isTerminated + " " + cell.mailbox.currentStatus + " "
                        + cell.mailbox.numberOfMessages + " " + cell.mailbox.systemDrain(SystemMessageList.LNil).size)
```

Reply:
So much better @raboof, thanks. I didn't want to touch Ordering and left as is but didn't like what that meant.